ablySpec/RealtimeClientConnection.swift

```diff
@@ -1357,12 +1362,12 @@ class RealtimeClientConnection: QuickSpec {
                                 done()
                             })
                         }
-
+                        
```

Comment:
Whitespace. 

Check this please:
<img width="401" alt="screen_shot_2016-02-23_at_20_17_37" src="https://cloud.githubusercontent.com/assets/3541185/13265388/97f51bbc-da6a-11e5-863a-8590df80b862.png">